src/test/helper.ts

```diff
@@ -0,0 +1,120 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { assert } from 'chai';
+
+export let doc: vscode.TextDocument;
+export let editor: vscode.TextEditor;
+
+// Default ansible configurations
+export const defaultAnsibleConfigurations = {
+  'ansible.useFullyQualifiedCollectionNames': true,
+  'ansibleLint.arguments': '',
+  'ansibleLint.enabled': true,
+  'ansibleLint.path': 'ansible-lint',
+  'ansibleNavigator.path': 'ansible-navigator',
+  'executionEnvironment.containerEngine': 'auto',
+  'executionEnvironment.enabled': false,
+  'executionEnvironment.image': 'quay.io/ansible/creator-ee:latest',
+  'executionEnvironment.pullPolicy': 'missing',
+  'python.activationScript': '',
+  'python.interpreterPath': 'python3',
+  'ansible.path': 'ansible',
+};
+
+/**
+ * Activates the redhat.ansible extension
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function activate(docUri: vscode.Uri): Promise<any> {
+  const extension = vscode.extensions.getExtension('redhat.ansible');
+  const activation = await extension?.activate();
+
+  try {
+    doc = await vscode.workspace.openTextDocument(docUri);
+    editor = await vscode.window.showTextDocument(doc, {
+      preview: true,
+      preserveFocus: false,
+    });
+    await vscode.languages.setTextDocumentLanguage(doc, 'ansible');
+
+    await sleep(5000); // Wait for server activation
```

Comment:
As of now, I don't have a better solution for this, since there's no such actionable indication when the server is initiated. Maybe, this can be discussed and a better implementation (if any), can be added in subsequent PRs.